cli/src/main/resources/help/server.adoc

```diff
@@ -0,0 +1,67 @@
+SERVER(1)
+=========
+:doctype: manpage
+
+
+NAME
+----
+server - server configuration and state management.
+
+
+DESCRIPTION
```

Comment:
description missing

Reply:
I added a description. hopefully it's alright.